docs/download/guide.md

```diff
@@ -538,7 +538,7 @@ Related terms and their definitions are given below:
 | _Compute Instance_  | A user virtual machine operating in a cloud environment.                                                                                                                                                                                                                                 |
 | _DACO_              | The Data Access Compliance Office which handles requests from researchers for access to controlled data from the ICGC.                                                                                                                                                                   |
 | _DACO Cloud Access_ | DACO access with supplemental approved Cloud Access status.                                                                                                                                                                                                                              |
```

Comment:
@hknahal maybe we change to something like this (had trouble rewording)?
`DACO approval now means automatic Cloud Access `